### PR TITLE
Feature/create files

### PR DESCRIPTION
**Pull request by iuriifaevskii**
- added main page
- added package json

*question*

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (Shooter75) [inline site/index.html]: This file looks very minimally.

File: site/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<html>
+<head>
+<title>Page Title</title>
```
Comment (Shooter75) [inline site/index.html]: **Title** isn't very suitable for this page!

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/iuriifaevskii/testproject.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (Shooter75) [inline package.json]: the license isn't correct for our project

**Comment by iuriifaevskii** [comment]
ok I'll fix it now

File: package.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/iuriifaevskii/testproject.git"
+  },
+  "author": "",
+  "license": "WBO",
```
Comment (Shooter75) [inline package.json]: Ти переграв в бокс, що ти таке пишеш?

**Comment by iuriifaevskii** [comment]
Oh I see this problem 
I need more time to fix it.

**Comment by iuriifaevskii** [comment]
Reopen because of new commit

**Pull request merged**